catalog/registry/catalog-registry-metacard-filter/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,32 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+
+<blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+    <reference id="filterBuilder"
+               interface="ddf.catalog.filter.FilterBuilder"/>
+    <reference id="filterAdapter"
+               interface="ddf.catalog.filter.FilterAdapter"/>
+
+    <bean id="registryMetacardFilter"
+          class="ddf.catalog.registry.metacard.filter.RegistryMetacardFilterPlugin">
+        <argument ref="filterBuilder"/>
+        <argument ref="filterAdapter"/>
+    </bean>
+
+    <!-- Register in the OSGi Service Registry -->
+    <service ref="registryMetacardFilter" interface="ddf.catalog.plugin.PreQueryPlugin" />
```

Comment:
You can just do `auto-export="interfaces"` instead of specifying the interface here. Then if the interface that you're implementing ever changes for one reason or another, you don't also need to remember to modify the blueprint file.